files/Aboot/boot0.j2

```diff
@@ -488,7 +488,12 @@ get_eeprom_value() {
 write_kdump_cmdline() {
     local sid="$(cmdline_get sid | sed 's/Ssd$//')"
     case "$sid" in
-        Wolverine*|Clearwater2*|OtterLake*|QuartzDd*|Redstart8Mk2Quartz4*|CitrineDd*)
+        OtterLake*)
+            if ! cmdline_has crashkernel; then
+                cmdline_add crashkernel=1G,high
```

Comment:
general question, is this 1G high applicable only for this platform. Should me make crashkernel allocation into high memory default ?

Also I see another PR from Arista : https://github.com/sonic-net/sonic-buildimage/pull/24382. May be we should close one of them ?